app/components/shared/environment_banner_component.rb

```diff
@@ -0,0 +1,8 @@
+class Shared::EnvironmentBannerComponent < ViewComponent::Base
```

Comment:
How do you feel about extending `GovukComponent::Base`?

Reply:
Sure! This was just copypasta'ed from another component 😅